tests/test_texture_array.py

```diff
@@ -166,19 +169,19 @@ def test_wide(test_graphic):
         buffer_shape=(2, 3),
         row_indices_size=2,
         col_indices_size=3,
-        row_indices_values=np.array([0, 8192]),
-        col_indices_values=np.array([0, 8192, 16384]),
+        row_indices_values=np.array([0, MAX_TEXTURE_SIZE]),
+        col_indices_values=np.array([0, MAX_TEXTURE_SIZE, 2 * MAX_TEXTURE_SIZE]),
```

Comment:
So if we have an data array of shape (1200, 2200), and the limit is 1024 then we need a `TextureArray` of shape (2, 3), that's the `buffer_shape` here.

for the rows, we need 2 because 1200 with a limit of 1024 means we will have array with 1024 rows and another with 176

Same logic for the columns and the other tests below for `test_tall` and `test_square`